tests/render/renderer/test_checkpoint_new_notebook_renderer.py

```diff
@@ -21,9 +21,16 @@ def test_find_datasource_with_asset_on_context_with_no_datasources(
 
 def test_find_datasource_with_asset_on_context_with_a_datasource_with_no_dataconnectors(
     titanic_pandas_data_context_with_v013_datasource_stats_enabled_with_checkpoints_v1_with_templates,
+    test_backends,
 ):
     context = titanic_pandas_data_context_with_v013_datasource_stats_enabled_with_checkpoints_v1_with_templates
     context.delete_datasource("my_datasource")
+
+    if any(
+        [dbms in test_backends for dbms in ["postgresql", "sqlite", "mysql", "mssql"]]
+    ):
+        context.delete_datasource("my_runtime_sql_datasource")
+
```

Comment:
Is it possible and would it make sense to loop through all the datasources and delete them? Might simplify this a bit

Reply:
I think we still want this since the "my_runtime_sql_datasource" would only exist if there were sqlalchemy backends